### DISCUSSION TITLE
`ERROR! Failed to call GitHub Status API!` on GitHub Actions

### DISCUSSION THREAD

**Discussion by eitsupi** [super-linter/super-linter#2641] [Q&A] [answered]
When I set up super linter, I get the following error message and cannot see where the error occurred. Is there any way to fix it?

```
ERROR! Failed to call GitHub Status API!
```

https://github.com/rocker-org/rocker-versioned2/runs/5548047135?check_suite_focus=true

**Comment by ferrarimarco** [comment, marked as answer]
Let's discuss in a single place: https://github.com/github/super-linter/issues/3137
